backend/OR-Tools/OR-Tools/OrToolsConverter.cs

```diff
@@ -55,5 +63,71 @@ public static Data ConvertToData(FileInput fileInput)
 
             return data;
         }
+
+        public FileOutput ConvertToFileOutput(ORSolver orSolver)
+        {
+            var fileOutput = new FileOutput();
+
+            RoutingDimension capacityDimension = orSolver.Routing.GetDimensionOrDie("Capacity");
+            RoutingDimension timeDimension = orSolver.Routing.GetMutableDimension("Time");
+
+            long totalLoad = 0, totalTime = 0, totalDistance = 0;
+
+            for (int index = 0; index < orSolver.Routing.Size(); ++index)
+            {
+                if (orSolver.Routing.IsStart(index) || orSolver.Routing.IsEnd(index))
+                {
+                    continue;
+                }
+                if (orSolver.Solution.Value(orSolver.Routing.NextVar(index)) == index)
+                {
+                    var node = orSolver.Manager.IndexToNode(index);
+                    fileOutput.DroppedLocation.Add(new Dropped { LocationName = locationsNames[node] });
+                }
+            }
+
+            Data data = new Data();
+            for (int i = 0; i < data.VehicleNumber; ++i)
+            {
+                Console.WriteLine("Route for Vehicle {0}:", i);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4619036)